ast/ast.go

```diff
@@ -145,9 +145,8 @@ type RecordSet interface {
 	Close() error
 }
 
-// ResultSetNode interface has ResultFields property which is computed and set by
-// optimizer.InfoBinder during binding process. Implementations include SelectStmt,
-// SubqueryExpr, TableSource, TableName and Join.
+// ResultSetNode interface has ResultFields property, represents a Node that returns result set.
```

Comment:
The ResultSetNode interface has a ResultFields property, representing a node that returns the result set.